src/photos/photoData.ts

```diff
@@ -0,0 +1,25 @@
+export interface TrimmedPhotoData {
+    id: number;
+    img_url: string;
+    sol: number;
+    earth_date: string;
+}
+
+
+export function trimPhotoData(photos: any[]): TrimmedPhotoData[] {
+    const trimmedPhotos: TrimmedPhotoData[] = [];
+
+    while(photos.length > 0) {
```

Comment:
Had some problems using for..in so I used this while. The for would iterate trough the indexes in the array and I think this is better. The popped data isn't used anywhere else. Might be a problem in larger programs.